pkg/agent/util/net_windows.go

```diff
@@ -371,3 +373,23 @@ func SetAdapterDNSServers(adapterName, dnsServers string) error {
 	}
 	return nil
 }
+
+// ListenUnix creates a listener on a unix domain socket or a Windows named pipe.
+// - If the specified address starts with "\\.\pipe\",  create a listener on the a Windows named pipe path.
+// - Else create a listener on a local unix domain socket.
+func ListenUnix(address string) (net.Listener, error) {
```

Comment:
It is strange to call it ListenUnix()? How about ListenLocalSocket/DialLocalSocket?

Reply:
Thanks Jianjun. It's better to use `ListenLocalSocket/DialLocalSocket`. Done